src/main/webapp/profile.html

```diff
@@ -23,7 +23,7 @@ <h1>Your Profile</h1>
         <div class="form-group row">
           <label for="name" class="col-sm-2 col-form-label">Name</label>
```

Comment:
Please bold or otherwise highlight the labels. This should probably be done as a CSS rule for all labels

Reply:
Ok l'll do this another PR so that it can be applied for all labels.